tests/graph/tests_loading.cpp

```diff
@@ -0,0 +1,19 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include <doctest/doctest.h>
+
+#include <rdf4cpp.hpp>
+
+using namespace rdf4cpp;
+
+TEST_CASE("load dataset") {
+    Dataset set{};
```

Comment:
For dataset we should load a nquads or trig file. It can be smaller than swdf.
But you can keep this test by changing it to `Graph`